neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -93,6 +171,54 @@ def test_wallet_close(self):
 
                 self.assertTrue(res)
 
+    def test_wallet_verbose(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
+            # test wallet verbose with no wallet
+            args = ['verbose']
+            res = CommandWallet().execute(args)
+            self.assertFalse(res)
+
+        # test wallet close with open wallet
+        self.OpenWallet()
+        args = ['verbose']
+        res = CommandWallet().execute(args)
+        self.assertTrue(res)
+
+    def test_wallet_migrate(self):
+        pass
+
+    def test_wallet_create_address(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
+            # test wallet create address with no wallet open
+            args = ['create_addr', 1]
+            res = CommandWallet().execute(args)
+            self.assertFalse(res)
+
+            self.OpenWallet()
+
+            # test wallet create address with no argument
+            args = ['create_addr']
+            res = CommandWallet().execute(args)
```

Comment:
not specifically at this line, but in the function it calls (not seeing that code here); 
Can you please give a more user friendly error message and/or capture the "not enough arguments" problem sooner? We now get

```
neo> wallet create_addr
int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
which is really confusing because you might try interpret this as it indicating that "create_addr" is read as an int() argument

Reply:
Amend of ^ 
I removed the early detection and checked the argument in `CommandWalletCreateAddress`.